api/src/handlers/chain_api.rs

```diff
@@ -106,9 +106,10 @@ impl OutputHandler {
 
 		let mut outputs: Vec<Output> = vec![];
 		for x in commitments {
-			if let Ok(output) = self.get_output(&x) {
-				outputs.push(output);
-			}
+			match self.get_output(&x) {
+				Ok(output) => outputs.push(output),
+				Err(e) => error!("{}", e),
```

Comment:
would you consider adding some context to the error message (here and below)?

Reply:
👍